recipe/build.sh

```diff
@@ -1,23 +1,16 @@
 #!/usr/bin/env bash
 set -euxo pipefail
 
+extra_args="-G Ninja"
 if test $(python -c "from __future__ import print_function; import platform; print(platform.system())") = Darwin ; then
   echo "Darwin"
-  extra_args="-DOPENMP_LIBRARIES=${CONDA_PREFIX}/lib -DOPENMP_INCLUDES=${CONDA_PREFIX}/include"
+  extra_args="$extra_args -DOPENMP_LIBRARIES=${CONDA_PREFIX}/lib -DOPENMP_INCLUDES=${CONDA_PREFIX}/include"
```

Comment:
btw @gfardell surely this is fine?

```suggestion
  extra_args="$extra_args -DOPENMP_ROOT=${CONDA_PREFIX}"
```

Reply:
Though STIR does have https://github.com/conda-forge/stir-feedstock/blob/main/recipe%2Fmeta.yaml#L46-L47

Reply:
continuing there